src/common/common.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+
+class SectionHandler:
+    """
+    A class encapsulating arbitrary XML element in the tree.
+    After conversion, it holds the JSON structure of its whole subtree.
+    """
+
+    def __init__(self):
+        self.json = {}
```

Comment:
Curiosity kills the 🐈 but: Is there a better name? `json` is not what is in there, right? Maybe defining `self.csaf = {}` would be clearer for the dict. Rationale: The variable holds the CSAF representation in the host language internal format (it will only later become json when dumped).

The later serialization to json would then also read better (IMO): `json.dumps(self.csaf)`

Looking at the source in tiny GitHub web UI snippets only, so maybe I miss the big picture.

In case the dict does not yet collect CSAF data but instead the model content and approximate structure, another option would be to define it as, well, `self.model = {}`

Reply:
Filip agreed on taking care about this one. Good point to have valuable namings.

Reply:
Implemented, thanks for valid feedback